docs/data_quality.md

```diff
@@ -0,0 +1,114 @@
+# Properties of a High Quality OSV Record
+
+## Version
+
+1.0.0 (SEMVER)
+
+## Purpose
+
+Describe the “good enough” OSV record that will be imported by OSV.dev
+
+### Out of scope
+
+This does not discuss the problem of record bit rot over time, after initial successful import. The problem of continuous revalidation and treatment of records that have been successfully imported will be dealt with separately in .
+
+Deferred to a future iteration: validating the existence of vulnerable functions in the `ecosystem_specific` field, if supplied.
+
+## Audience
+
+1. Current and aspiring OSV record producers
+2. Downstream OSV.dev record consumers
+
+## Rationale
+
+OSV.dev seeks to be an comprehensive, accurate and timely database of known vulnerabilities (that is highly automation friendly). In order to meet this accuracy goal, a quality bar needs to be both defined and sustainably enforced.
+
+## Properties of a High Quality OSV Record
+
+### Valid
+
+As a prerequisite, it is assumed that a record passes [JSON Schema validation](#appendix-a-osv-schema-validation) for the version of the OSV Schema it declares itself to comply with in the `schema_version` field, or 1.0.0 if it does not.
+
+### Precise
+
+A high quality OSV record allows a consumer of that record to be able to answer the following questions in an **automated** way, at scale:
+
+* “Does this vulnerability, as described, impact me?
+  * “What version do I need to upgrade to for it not to impact me?” 
+
+The definition of “impact” will vary depending on how fine-grained the information available is (i.e. package-level or symbol-level for software library packages). Package-level precision is the minimum standard.
+
+#### Properties
+
+* for version and commit ranges
+  * `affected[]`.`ranges[]`.`introduced` is defined
+  * prefer `affected[]`.`ranges[]`.`fixed` over `affected[]`.`ranges[]`.`last_affected`
+    * this minimizes false negatives
+  * distinct ranges for `introduced..fixed` and/or `introduced..last_affected` *(i.e. introduced and fixed commits can't be the same)*
+  * values in `introduced` are before/less than `fixed`/`last_affected`
+* for version (`ECOSYSTEM` and `SEMVER`) ranges
+  * the versions exist in the specific package ecosystem
+* for commit (`GIT`) ranges
+  * the commits exist in the specified `repo` *(i.e. they are not from another GitHub fork)*
```

Comment:
These two parent bullets had me confused since they read like `case` statements with the one about `for version and commit ranges`. Really the latter two are a subset of the first parent bullet. Could we indent them since they are all related to the version ranges covered by the first bullet point?
```suggestion
  * for version (`ECOSYSTEM` and `SEMVER`) ranges
    * the versions exist in the specific package ecosystem
  * for commit (`GIT`) ranges
    * the commits exist in the specified `repo` *(i.e. they are not from another GitHub fork)*
```